content/docs/state-and-lifecycle.md

```diff
@@ -155,23 +155,23 @@ Note how we pass `props` to the base constructor:
   }
 ```
 
-Class components should always call the base constructor with `props`.
+Les composants à base de classe doivent toujours appeler le constructeur de base avec `props`.
 
-3) Remove the `date` prop from the `<Clock />` element:
+3) Supprimez la prop `date` de l'élément `<Horloge />` :
 
 ```js{2}
 ReactDOM.render(
-  <Clock />,
+  <Horloge />,
   document.getElementById('root')
 );
 ```
 
-We will later add the timer code back to the component itself.
+Nous rajouterons plus tard le code du minuteur dans le composant lui-même.
 
-The result looks like this:
+Le résultat est tel que suit :
```

Comment:
```suggestion
Le résultat ressemble à ceci :
```

Woaaah, « est tel que suit » ! On est dans les travées de l'Académie, là ! C'est très, très, très formel comme ton, on utilise beaucoup plus « ressemble à ceci » pour tous les « looks like this », ne serait-ce que par préservation du ton d'origine.

Reply:
En effet c'est plus léger 😅